config/routes.rb

```diff
@@ -2,6 +2,8 @@
   devise_for :users
   resources :users
   resources :questions
+  resources :answers, defaults: { format: 'js' }
   get '/' => 'welcome#index'
+  get '/answers/:id', to: 'answers#show', defaults: { format: 'js' }
```

Comment:
@o-volosianskyi what's wrong with `resources :answers` that you need another route?